### PR TITLE
Nilearn sprint example consolidation spheres

### PR DESCRIPTION
**Pull request by illdopejake**
At @KamalakerDadi 's request, I merged examples 8.4.5 + 8.4.8 (see issue #1986 ). This mostly involved stacking them and removing some redundancies.

I also noticed that 5 subjects were downloaded, but only one was used. So I changed n_subjects=1.

Finally, I fixed a few links pointing to notebook 8.4.5 (none I could find pointed to 8.4.8).

### CODE REVIEW

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 1]
```diff
@@ -15,87 +14,217 @@
...
 
-###############################################################################
-# Load fMRI data and Power atlas
-# ------------------------------
+##########################################################################
+# Retrieve the brain development fmri dataset
+# -------------------------------------------
 #
-# We are going to use a single subject from the movie watching fmri dataset.
+# We are going to use subjects from the movie watching fmri dataset.
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
# We are going to use a subject from the a development fmri dataset.
```

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 2]
```diff
@@ -15,87 +14,217 @@
...
+##############################################################################
+# 3D visualization in a web browser
+# ---------------------------------
+# An alternative to :func:`nilearn.plotting.plot_connectome` is to use
+# :func:`nilearn.plotting.view_connectome`, which gives more interactive
+# visualizations in a web browser. See :ref:`interactive-connectome-plotting`
+# for more details.
+
+
+view = plotting.view_connectome(partial_correlation_matrix, dmn_coords)
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
view = plotting.view_connectome(partial_correlation_matrix, dmn_coords)
# In the jupyter notebook, the following line will display the view
view
```

Comment (GaelVaroquaux) [reply examples/03_connectivity/plot_sphere_based_connectome.py]: Sorry, ignore that.

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 3]
```diff
@@ -15,87 +14,217 @@
...
+# First we fetch the coordinates of the Power atlas
 
-###############################################################################
-# We fetch the coordinates of Power atlas.
 power = datasets.fetch_coords_power_2011()
 print('Power atlas comes with {0}.'.format(power.keys()))
 
+
+#########################################################################
+# .. Note ::
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
# .. note::

```

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 4]
```diff
@@ -15,87 +14,217 @@
...
+#     You can retrieve the coordinates for any atlas, including atlases
+#     not included in nilearn, using :func: `nilearn.plotting.find_parcellation_cut_coords`.
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
#     not included in nilearn, using :func:`nilearn.plotting.find_parcellation_cut_coords`.
```

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 5]
```diff
@@ -106,10 +235,14 @@
 plotting.plot_connectome(matrix, coords, title='Power correlation graph',
                          edge_threshold='99.8%', node_size=20, colorbar=True)
 
+
 ###############################################################################
-# Note the 1. on the matrix diagonal: These are the signals variances, set to
-# 1. by the `spheres_masker`. Hence the covariance of the signal is a
-# correlation matrix
+# .. Note ::
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
# .. note:: 
```

File: examples/03_connectivity/plot_sphere_based_connectome.py [hunk 6]
```diff
@@ -106,10 +235,14 @@
...
+
 ###############################################################################
-# Note the 1. on the matrix diagonal: These are the signals variances, set to
-# 1. by the `spheres_masker`. Hence the covariance of the signal is a
-# correlation matrix
+# .. Note :: 
+# 
+#     Note the 1. on the matrix diagonal: These are the signals variances, set to
+#     1. by the `spheres_masker`. Hence the covariance of the signal is a
+#     correlation matrix
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_sphere_based_connectome.py]:
```suggestion
#     correlation matrix.
```

**Comment by GaelVaroquaux** [comment]
I did a pass and that's all I found. Hopefully this builds cleanly and that it's ready for merge!

**Comment by illdopejake** [comment]
merging to PR #1986 to reduce CircleCI bandwidth

**Pull request closed without merging**